### PR TITLE
M8

### PR DESCRIPTION
**Pull request by levant47**

### CODE REVIEW

File: CatalogService/BLL/ItemService.cs [hunk 1]
```diff
@@ -41,7 +41,13 @@ public async Task<ItemEntity> Update(int id, ItemUpdateDto update)
         await ValidateItem(update);
         await _unitOfWork.Start();
         await _repository.Update(id, update);
-        await _messagingService.Save(new ItemUpdatedMessage { ItemId = id, Update = update });
+        await _messagingService.Save(new ItemUpdatedMessage
+        {
+            TraceId = Activity.Current?.TraceId.ToString(),
+            SpanId = Activity.Current?.SpanId.ToString(),
+            ItemId = id,
```
Comment (rkosmirak) [inline CatalogService/BLL/ItemService.cs]: I don't think that setting Trace and Span Id is a responsibility of the Business Layer. If you would decide to send new message you would need to don't forget to set them. Please, move this logic to the MessagingService Instead

File: CatalogService/BLL/Usings.cs [hunk 2]
```diff
@@ -1 +1 @@
-﻿global using System.Data;
+﻿global using System.Diagnostics;
```
Comment (rkosmirak) [inline CatalogService/BLL/Usings.cs]: Not sure that we need this namespace in Business layer

File: CatalogService/BLL/ItemService.cs (hunk 1 above)
Comment (levant47) [reply CatalogService/BLL/ItemService.cs]: Fixed

File: CatalogService/BLL/Usings.cs (hunk 2 above)
Comment (levant47) [reply CatalogService/BLL/Usings.cs]: Fixed

**Review by rkosmirak — APPROVED** [review]

**Pull request merged**